platform.md

```diff
@@ -122,6 +136,11 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
+## OCI Image
+The OCI image containing the compiled application is generated by the Build Image during the [Export Phase](#export-phase).
+
+If the `/cnb/lifecycle/exporter` is invoked with the optional `-source-metadata` argument, the lifecycle MUST add a `io.buildpacks.project` image config label to the new OCI image with a [value as described in the data-format](#io.buildpacks.app.source-oci-image-label-json) section.
```

Comment:
We want `io.buildpacks.project` to cover more than just source metadata, so I think this file should be called something like `project-metadata.toml` instead. (But not `project.toml`, which is already proposed for the project descriptor.)

Reply:
Should this be a more general `-metadata` or `-project-metadata` flag instead of something specific to source?

Maybe we actually do want to pass in a file with the same schema as `project.toml`. After all, I can't see `project.toml` having a different structure for source than what's defined here.

Reply:
I add my vote for `--project-metadata` flag and `project-metadata.toml` filename.

> Maybe we actually do want to pass in a file with the same schema as project.toml. After all, I can't see project.toml having a different structure for source than what's defined here.

I think it would be best to keep them separate. This current schema is really open-ended in regards to how the data is formatted whereas `project.toml` is more specific. For example, the proposed use cases in the RFC call out specific dynamic pieces of information such as `tags` which you wouldn't want to hardcode into the `project.toml`.

Reply:
We've renamed to `-project-metadata` and `project-metadata.toml` in the latest commit.